openmc/mgxs/mgxs.py

```diff
@@ -212,11 +229,70 @@ def num_groups(self):
         return self.energy_groups.num_groups
 
     @property
+    def scores(self):
+        return ['flux', self.rxn_type]
+
+    @property
+    def filters(self):
+        group_edges = self.energy_groups.group_edges
+        energy_filter = openmc.Filter('energy', group_edges)
```

Comment:
Oh....yes good point. Do you see any reason to avoid this? Of course it
would be simple to store the filters in a new class attribute if necessary.
On May 13, 2016 12:43 PM, "Paul Romano" <notifications@github.com> wrote:

> In openmc/mgxs/mgxs.py
> <https://github.com/mit-crpg/openmc/pull/644#discussion_r63214135>:
>
> > @@ -212,11 +229,70 @@ def num_groups(self):
> >          return self.energy_groups.num_groups
> >
> >      @property
> > +    def scores(self):
> > +        return ['flux', self.rxn_type]
> > +
> > +    @property
> > +    def filters(self):
> > +        group_edges = self.energy_groups.group_edges
> > +        energy_filter = openmc.Filter('energy', group_edges)
>
> Oh sorry, that was confusing. What I meant is that [id(f) for f in
> mgxs_object.filters] will return a different list each time you evaluate
> it.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-crpg/openmc/pull/644/files/47ef320ad517612376e181ec6a6bc42ca0db98ce#r63214135>
>